sites/public/pages/finder.tsx

```diff
@@ -172,10 +203,12 @@ const Finder = () => {
                 </div>
 
                 <div className="bg-gray-300 flex flex-row-reverse justify-between py-8 px-20">
-                  {currentStep === Object.keys(formData).length ? (
-                    <Button type="submit" styleType={AppearanceStyleType.primary}>
-                      {t("t.submit")}
-                    </Button>
+                  {questionIndex === formData.length - 1 ? (
+                    <>
```

Comment:
🧹

Reply:
@emilyjablonski I found that rendering the submit button without a tag wrapping it resulted in it effectively being clicked which skips the disclaimer page. Do you know of a cleaner way around this issue?

Reply:
With some further research, I found this [discussion on it](https://stackoverflow.com/questions/71754898/involuntary-form-submit-when-rendering-submit-button-in-reactjs). Adding a key also solved the issue and prevented the auto-submit. Do you have a preference between these solutions?